browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -257,7 +233,7 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
         Block trackers &amp; ads
       </message>
       <message name="IDS_BLOCK_TRACKERS_ADS_SUMMARY" desc="Text summary for Block trackers and ads.">
-        Prevents ads, popups, and trackers from loading
```

Comment:
Not sure what warning is fixed by this change.

Reply:
Not a warning fix, i realised the text wasn't complete as per the design specs